book/website/collaboration/ppie-events.md

```diff
@@ -31,26 +31,29 @@ It covers:
 
-This chapter is focused in particular on how to make research events more inclusive and accessible to members of the public and people with lived experience who do not have experience or a background as a researcher, as they can face particular barriers to participation.
+This chapter is focused in particular on how to make research events more inclusive and accessible to members of the public and people with lived experience who do not have experience or a background as a researcher, as they can face particular barriers to participation. 
 
-Many research projects are increasingly involving members of the public in the research. 
+Many research projects are increasingly including members of the public in the research process. 
 Within the realm of biomedical and care-related research, this is often referred to as "Public and Patient Involvement and Engagement" or "PPIE" which has a goal to actively involve people with the lived experience of a specific disability, situation, health condition, or experience in the research process. {cite:ps}HDRUK2023ppie 
-PPIE is done to ensure that research is aligned with the needs of those affected by it, and can also improve the quality of research by including lived perspectives. 
-As part of this involvement and engagement, researchers and research staff will need to plan events that are inclusive and accessible for people with lived experience, and delivered in a way that supports people with lived experience taking part equitably.
+PPIE is done both to ensure that research is aligned with the interests of people affected by it, and to improve the quality of research by including lived knowledge and perspectives. 
+As part of this involvement and engagement, researchers and research staff will need to plan events that are inclusive and accessible for people with lived experience, and delivered in a way that avoids prsenting barriers topeople with lived experience taking part equitably.
 
 "Lived experience" can be a range of things such as a disability, long-term health condition, bereavement, having worked in a specific profession, or the lived experience of caring responsibilities. 
-In the context of event planning, any of these people with lived experiences might have slightly different accommodation needs to support their engagement at your event. 
-Thus, your considerations need to cover both the potential disability requirements of your attendees as well as the specific barriers that would be experienced by your attendees with lived experience. 
+In the context of event planning, there are practices we can use to foster participation of this diverse group of people.
+
+Thus, your considerations need to cover both the potential disability-related barriers experienced by your attendees as well as the barriers that would be experienced by your attendees with lived experiences such as parental responsibilities, cultural differences, or minority status. 
```

Comment:
Is there a reason we've narrowed down lived experience to just those 3 examples?